doc/tutorials/Launching-Services.md

```diff
@@ -15,7 +15,7 @@ Launching packages this way will allow you to verify if the package is correctly
 ## Launching Services in Docker
 Most services can be run in the 'full' container, however detectors often come with conflicting requirements. Therefore we choose to create seperate images for these modules, and run them in their own images.  
 If you want to run a detector locally, use the same command but in the corresponding container. For example, for the stt package, use the ros_w2l container for running the script.  
-Similarly, if there are seperate computers controlling the robot hardware, we may wish to have a seperate image for running on that computer. This is often named 'harmoni_hardware'.
+Similarly, if there are seperate computers controlling the robot hardware, you may wish to have a seperate image for running on that computer. This is often named 'harmoni_hardware'.
 
 
 ### Roslaunch Aliases
```

Comment:
If we are including these in the docs they should be tested to make sure they still work (I think some are outdated)

Reply:
Agreed. I wasn't really sure what to do with it and was unsure about removing things...a lot of the tutorials need a review and possible rework.